src/audio/volume/Kconfig.hifi

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+comment "volume optimization level select"
+
+choice "VOLUME_HIFI_LEVEL_SELECT"
+	prompt "choose which hifi level used for volume module"
+
+	config VOLUME_DEFAULT_OPT_LEVEL
+		prompt "dummy optimized option for volume module"
+		bool
+		help
+			This option is dummy option for volume module, when this was
+			selected, optimization level will be determined by compiler.
+
+	config VOLUME_HIFI3
+		prompt "choose hifi3 intrinsic optimized volume module"
```

Comment:
also needs to depend on ARCH_XTENSA && ARCH_HAS_HIFI3 (can be defined in SOC Kconfig), same for the other Hifi levels. These will both come from Zephyr, but I've no issue adding this PR now whilst the Zephyr Kconfigs are WIP for HiFi and then updating when Zephyr is merged.